packages/overlay/test/position-mixin.common.js

```diff
@@ -93,6 +93,18 @@ describe('position mixin', () => {
     expect(overlay.$.overlay.scrollTop).to.equal(100);
   });
 
+  it('should close overlay if element is hidden', async () => {
```

Comment:
nit: The tests aren't specifically testing the case described in the issue, where the overlay is first open and then the target is hidden